samples/nrf9160/http_application_update/src/main.c

```diff
@@ -27,13 +27,14 @@ void bsd_recoverable_error_handler(uint32_t err)
 	printk("bsdlib recoverable error: %u\n", err);
 }
 
+#define NO_TLS -1
```

Comment:
I suggest putting #defines in a header file or at the top of this file.

Reply:
Put it into a Kconfig variable